www/docs/docusaurus.config.js

```diff
@@ -54,6 +55,19 @@ const config = {
       theme: require("prism-react-renderer/themes/vsDark"),
       darkTheme: require("prism-react-renderer/themes/vsDark"),
     },
+    zoom: {
+      selector: '.markdown img',
+      background: {
+        light: 'rgb(255, 255, 255)',
+        dark: 'rgb(50, 50, 50)'
+      },
```

Comment:
According to the documentation this should be inside the `config` object

Reply:
Yes, the documentation is not updated. I had tried these `background` properties inside the `config` object and no changes were being reflected.
This is the thread which I referred: https://github.com/gabrielcsapo/docusaurus-plugin-image-zoom/issues/10